docs/scos/dev/glue-api-guides/202311.0/conventions/what-is-api-convention.md

```diff
@@ -0,0 +1,44 @@
+---
+title: What is a Glue API convention?
+description: This document will describe what is convention in the Glue API and how it impacts your resources.
+last_updated: December 12, 2023
+template: howto-guide-template
+---
+
+Convention defines how request and response should look like in order to use something that is agreed in the industry and focus on your application rather than on response format. Using convention allows to users that are not familiar with your API, to start to use it faster as the way how to use it is familiar to them.
+
+## Using API conventions in Spryker
+
+Spryker is using conventions for its API application. But existing Glue API application using them a bit differently.
+Legacy Glue API is tightly coupled with a [JSON:API](https://jsonapi.org/) convention. So all your endpoints will follow this convention in any case and you have no control over it.
```

Comment:
You introduce a new phrase here "Legacy Glue" What is it? provide more context or references please.